besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -2073,6 +2110,31 @@ public void rpcHttpOptionsRequiresServiceToBeEnabled() {
     assertThat(commandErrorOutput.toString(UTF_8)).isEmpty();
   }
 
+  @Test
```

Comment:
Don't see any tests for environment variables, can test be added for that as well